monailabel/endpoints/datastore.py

```diff
@@ -247,12 +247,14 @@ async def api_put_label_info(label: str, tag: str, info: str = Form("{}"), user:
 
 # This shall be deprecated and use above ones to update the label info
 @router.put("/updatelabelinfo", summary="Update label info")
-async def api_update_label_info(label: str, params: str = Form("{}"), user: User = Depends(get_annotator_user)):
-    return update_label_info_deprecated(label, params)
+async def api_update_label_info(
+    label: str, tag: str, params: str = Form("{}"), user: User = Depends(get_annotator_user)
+):
+    return update_label_info_deprecated(label, tag, params)
```

Comment:
i think you can call `return update_label_info(label, tag, info, user.username)` and remove `update_label_info_deprecated` function